src/Sylius/Component/Mailer/Event/EmailRenderedEvent.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Mailer\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * @author Jérémy Leherpeur <jeremy@leherpeur.net>
+ */
+class EmailRenderedEvent extends Event
```

Comment:
Why having two event classes that holds almost same data instead of one, and some optional parameter while using  "recipients"?